autogluon/utils/tabular/ml/models/lgb/hyperparameters/parameters.py

```diff
@@ -32,7 +32,6 @@ def get_param_multiclass_baseline_custom(num_classes):
         'num_boost_round': DEFAULT_NUM_BOOST_ROUND,
         'num_threads': -1,
         'objective': 'multiclass',
-        'metric': 'multi_error,multi_logloss',
```

Comment:
Since 'metric' is now removed from this file, can we add comment in this file that users can also specify additional hyperparameters listed here: 
https://lightgbm.readthedocs.io/en/latest/Parameters.html#

I think this should be done in both this file and the other models' hyperparameters/ files (especially catboost), otherwise users do not know which hyperparameters are even available for them to play with (since we have no documentation of this anywhere outside of these files).

Reply:
Agreed, will do in a separate PR dedicated to documentation of model hyperparameters